Sheikh_Shahzeb/assignment1_project/project_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'project'
+RSpec.describe Project do  
+  # ...
+  # building off our previous example lets add a test to make sure our method returns the correct value
+  it 'has a method elevator_pitch to explain name and description' do
+
+    project1 = Project.new('Project 1', 'description 1')
+    #if we call elevator_pitch method we should expect to get that project name and description back
+    expect(project1.elevator_pitch).to eq("project name is #{project1.project_name} and project description is #{project1.project_description}")
```

Comment:
since you're testing you should make the .to eq statement to be literal instead of using interpolation.  It should show what you actually expect i.e. expect(project1.elevator_pitch).to eq("project name is Project 1 and project description is description 1")